pywbem/_subscription_manager.py

```diff
@@ -541,6 +557,22 @@ def add_listener_destinations(self, server_id, listener_urls, owned=True):
             permanent. See :ref:`WBEMSubscriptionManager` for details about
             these ownership types.
 
+          persistence_type (:class:`py:int` or None):
+            The PersistenceType property defines for the WBEM server
+            persistence of the destination instance as follows: 1-Other, 2-
+            Permanent, 3-Transient. Most WBEM servers set the value to
+            2(Permanent) if no value is provided with a new instance. pywbem
+            does not support the Other value because it does not support the
+            OtherPersistenceType property.
+
+            If persistence_type is None, the PersistenceType property is
+            created with value 3(Transient) for owned destinations; it is not
+            created for permanent destinations.
```

Comment:
Should we then not set 3 as the default right away, instead of taking the detour through None?

Reply:
This could go either way since there is no real single default. If the parameter is not provided, we set to 3 for owned subscriptions and leave it as None so the Property is not created for permanent subscriptions.

Modified the text slightly to make it clearer.

Reply:
I did not realize the detail that the default depends on ownership.